main/rpc/index.js

```diff
@@ -228,6 +229,18 @@ const rpc = {
   resolveAragonName (name, chainId, cb) {
     resolveName(name, chainId).then(result => cb(null, result)).catch(cb)
   },
+  async resolveEnsName (name, cb) {
+    try {
+    log.info('resolving ENS name ', {name})
+    const nebula = nebulaApi()
+    const {addresses: {eth: ethAddress}} = await nebula.ens.resolve(name)
+    return cb(null, ethAddress)
+    }
+    catch(err) {
+      log.debug(`Could not resolve ENS name ${name}`, err)
```

Comment:
this one should actually be a higher level, maybe `warn`  because it means something went wrong
```suggestion
      log.warn(`Could not resolve ENS name ${name}`, err)
```